api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,26 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the *x-client-trace-id*
+    // header is set. Must be an number between 0 and 100; resolution is to the nearest 1% (rounded
+    // down). Defaults to 100. This variable is a direct analog for the variable of the same name in
+    // the `HTTP Connection Manager
+    // <https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/runtime.html?highlight=sample>`_.
+    Percent client_enabled = 3;
+
+    // Global target percentage of requests that will be traced after all other checks have been
+    // applied (force tracing, sampling, etc.). Must be a number between 0 and 100; resolution is to
+    // the nearest 1% (rounded down). Defaults to 100. This variable is a direct analog for the
+    // variable of the same name in the `HTTP Connection Manager
+    // <https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/runtime.html?highlight=sample>`_.
+    Percent global_enabled = 4;
+
+    // Global target percentage of requests that will be randomly traced. Must be a number between 0
+    // and 100; resolution is to the nearest 0.01%, rounded down. Defaults to 100. This variable is
+    // a direct analog for the variable of the same name in the `HTTP Connection Manager
+    // <https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/runtime.html?highlight=sample>`_.
```

Comment:
You can do `:ref:` style internal links to avoid relying on possible stale external links, grep around for how `:ref:` is used to see how.

Reply:
Fixed... _I think_